### PR TITLE
New package: f3d-2.5.0

### PR DESCRIPTION
**Pull request by Jipok**
Continue https://github.com/void-linux/void-packages/pull/46014

### CODE REVIEW

**Comment by abenson** [comment]
Please squash these into a single "New package" commit.

**Comment by Jipok** [comment]
Done

File: srcpkgs/f3d/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'f3d'
+pkgname=f3d
+version=2.5.0
+revision=1
+build_style=cmake
+makedepends="vtk-devel"
+depends="vtk"
```
Comment (abenson) [inline srcpkgs/f3d/template:7]: This isn't necessary, the dependency will be automatically detected.

Comment (Jipok) [reply srcpkgs/f3d/template:7]: image

Comment (abenson) [reply srcpkgs/f3d/template:7]: Sorry, I should've clarified: I meant the `depends`, not the `makedepends`.

**Comment by Jipok** [comment]
¯_(ツ)/¯

**Pull request closed without merging**